csharp/Urban.DCP.Data/UserHelper.cs

```diff
@@ -89,7 +89,10 @@ public static IList<SecurityRole> GetUserRoles(string userName)
                 userAccount.EmailConfirmed = confirmedEmail.Value;
             }
 
-            userAccount.Affiliation = affiliation;
+            if (affiliation != null)
```

Comment:
There's no place in the UI to do that at present, but I did leave it as a pure `null` check as opposed to a `nullOrEmpty` so that a `""` would effectively clear it out.